cmd/install.go

```diff
@@ -91,13 +98,46 @@ func installRun(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func checkCRDExistence(k8sClient client.Client, crdName, documentationURL string) error {
+	crd := &apiextensionsv1.CustomResourceDefinition{}
+	err := k8sClient.Get(context.Background(), types.NamespacedName{Name: crdName}, crd)
+	if err != nil {
+		logf.Log.Info(fmt.Sprintf("CRD %s not found. Please visit %s for installation instructions.", crdName, documentationURL))
+		return fmt.Errorf("dependency CRD %s is not installed. Please follow the installation guide at %s", crdName, documentationURL)
+	}
+	return nil
+}
+
+func performDependencyChecks(k8sClient client.Client) error {
+	var dependencyErrors []error
+
+	// Perform each check and collect errors
+	if err := checkCRDExistence(k8sClient, "gateways.networking.istio.io", "https://istio.io/latest/docs/setup/additional-setup/getting-started/"); err != nil {
```

Comment:
Maybe we should just link to https://docs.kuadrant.io/getting-started-single-cluster/#prerequisites instead...

Reply:
As it stands, the dependency error instructs to 'Please follow the installation guide at..'
If that is and always will be the same as what other docs (like on docs.kuadrant.io) will say, then that seems like a good dev exp as it brings them closer to the solution quicker than having to parse through the prerequisites section of guide that may have multiple prerequisites.

If however it may be different, like specifying a specific version or a specific way of installing, then linking to docs.kuadrant.io is the safest.

Reply:
I like your point about getting people to the right information, sooner. I think these looks are generic/non-versioned enough to be quite stable.